src/Schema/Domain/Content/Mapper/FieldDefinition/DecoratingFieldDefinitionMapper.php

```diff
@@ -35,6 +37,16 @@ public function mapToFieldValueResolver(FieldDefinition $fieldDefinition): ?stri
         return $this->innerMapper->mapToFieldValueResolver($fieldDefinition);
     }
 
+    public function mapToFieldValueInputType(ContentType $contentType, FieldDefinition $fieldDefinition): ?string
+    {
+        /** @deprecated this test will be removed in ezplatform-graphql 2.x */
+        if ($this->innerMapper instanceof FieldDefinitionInputMapper) {
+            return $this->innerMapper->mapToFieldValueInputType($contentType, $fieldDefinition);
+        }
+
+        throw new LogicException('The inner mapper is not a FieldDefinitionInputMapper. This method should not have been called in an 1.x versions of ezplatform-graphql.');
```

Comment:
How about:
```suggestion
        throw new LogicException('The inner mapper is not a FieldDefinitionInputMapper. This method should not be called in ezplatform-graphql 1.x.');
```

Reply:
The passive phrasing was because it is not called by the code user, but by the code itself. A developer can't call that directly.